app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parameters = command.split()
+    if len(command.split()) == 3:
+        parameters = command.split()
```

Comment:
what is the sense of it?
```suggestion
```

Reply:
The check for the length should be present. Instead you shouldn't duplicate command.split() call. Save the result of this call to a variable and then use unpacking